src/animals/Cat.java

```diff
@@ -1,10 +1,13 @@
 package animals;
 
 import java.util.Random;
-import veterinary.Vet;
 
 public class Cat extends Animal {
 	
+	/**
+	 * CatBreed enum catBreed is defined for the Cat Class. It is used in the Cat constructor.
```

Comment:
the comment needs to go above the relevant piece of code. Not in one block. Move the cat name description to above catName. You are also describing the CatBreed enum here, not the variable catBreed. The enum description should go with the enum definition.

Reply:
changed for enum description to be above catBreed; catName description above catName[]